multimodal/src/autogluon/multimodal/data/datamodule.py

```diff
@@ -58,6 +59,10 @@ def __init__(
         id_mappings
              Id-to-content mappings. The contents can be text, image, etc.
              This is used when the dataframe contains the query/response indexes instead of their contents.
+        val_is_train
+             Whether we want to treat validation as training.
+             This is used when we want to use val_loss as val metric, and thus training_pipeline
+             instead of val_pipeline during validation.
```

Comment:
I think we can say `instead of the testing pipeline during validation`?

Reply:
Modified the docstring.